domains/certificates/Query.API/API.IntegrationTests/IssuingContractCleanup/IssuingContractCleanupTests.cs

```diff
@@ -88,4 +85,10 @@ public async Task ShouldOnlyDeleteExpiredIssuingContracts()
         contracts.Select(x => x.Id).Should().Contain(nullEndDateContract.Id);
         contracts.Select(x => x.Id).Should().Contain(endDateContract.Id);
     }
+
+    private ApplicationDbContext GetDbContext()
+    {
+        var contextOptions = new DbContextOptionsBuilder<ApplicationDbContext>().UseNpgsql(postgresContainer.ConnectionString).Options;
+        return new ApplicationDbContext(contextOptions);
```

Comment:
Wont we benefit of setting the using statement here so we dont forget it? eg. line `28` :)

Reply:
Nope, because the using statement will dispose the object when it goes out of scope. The scope in this case is the method, however the object is returned from the method. It is therefore the responsibility of the caller to dispose the object.